openedx/core/djangoapps/oauth_dispatch/jwt.py

```diff
@@ -138,18 +137,7 @@ def _get_use_asymmetric_key_value(is_restricted, use_asymmetric_key):
     """
     Returns the value to use for use_asymmetric_key.
     """
-    # TODO: (ARCH-162)
-    # If JWT scope enforcement is enabled, we need to sign tokens
-    # given to restricted applications with a key that
-    # other IDAs do not have access to. This prevents restricted
-    # applications from getting access to API endpoints available
-    # on other IDAs which have not yet been protected with the
-    # scope-related DRF permission classes. Once all endpoints have
-    # been protected, we can enable all IDAs to use the same new
-    # (asymmetric) key.
```

Comment:
Should some of this documentation about when we need to use the asymmetric key end up in the docstring? It seems good to record that somewhere.

Reply:
It's all outdated documentation, which is what makes things confusing, and is one of the reasons why I want to remove all of this. :)